src/views/EAAbout/components/CardProjects/CardProjects.tsx

```diff
@@ -1,94 +1,80 @@
-          buttonType={ButtonType.Primary}
-          widthText="100%"
-          label="View Projects"
-        />
-      </Card>
-    </Container>
-  );
-};
+    <CardProject>
+      <CardDescription>View all the of the projects {actorName} is involved in and there status.</CardDescription>
+      <InternalLinkButtonStyled label="View Projects" href={siteRoutes.ecosystemActorProjects(shortCode)} showIcon />
+    </CardProject>
+  </Container>
+);
 
 export default CardProjects;
 
-const Container = styled.div({
+const Container = styled('div')({
   display: 'flex',
   flexDirection: 'column',
+  gap: 8,
   width: '100%',
-  marginBottom: 32,
 });
 
-const Title = styled.h3<WithIsLight>(({ isLight }) => ({
-  margin: '0 0 16px',
-  color: isLight ? '#231536' : '#D2D4EF',
-  fontSize: 24,
-  fontWeight: 600,
-  lineHeight: 'normal',
+const Title = styled('h3')(({ theme }) => ({
+  fontFamily: 'Inter, sans-serif',
+  margin: '0 0 0px',
+
+  color: theme.palette.isLight ? theme.palette.colors.gray[900] : theme.palette.colors.gray[50],
+  fontSize: 20,
+  fontWeight: 700,
+  lineHeight: '24px',
   letterSpacing: 0.4,
 }));
 
-const Card = styled.div<WithIsLight>(({ isLight }) => ({
+const CardProject = styled(Card)(({ theme }) => ({
   display: 'flex',
-  padding: '16px 16px 24px 16px',
+  padding: 8,
   flexDirection: 'column',
   alignItems: 'flex-start',
-  borderRadius: 6,
-  backgroundColor: isLight ? '#FFFFFF' : '#10191F',
-  boxShadow: isLight
-    ? '0px 1px 3px 0px rgba(190, 190, 190, 0.25), 0px 20px 40px 0px rgba(219, 227, 237, 0.40)'
-    : '10px 15px 20px 6px rgba(20, 0, 141, 0.1)',
+  gap: 24,
+
+  backgroundColor: theme.palette.isLight ? '#FFFFFF' : theme.palette.colors.charcoal[900],
+  boxShadow: theme.palette.isLight ? theme.fusionShadows.shortShadow : theme.fusionShadows.darkMode,
+  [theme.breakpoints.up('desktop_1440')]: {
+    padding: 16,
+    gap: 16,
+  },
 }));
 
-const CardDescription = styled.p<WithIsLight>(({ isLight }) => ({
-  margin: '0 0 24px',
-  color: isLight ? '#546978' : '#9FAFB9',
+const CardDescription = styled('p')(({ theme }) => ({
+  margin: '0 0 0px',
+  color: theme.palette.isLight ? '#546978' : theme.palette.colors.gray[500],
   fontSize: 15,
   fontWeight: 500,
   lineHeight: '24px',
   letterSpacing: 0.4,
 }));
 
-const ViewAllBtn = styled(LinkButton)({
-  textAlign: 'center',
-  borderRadius: '22px',
-  height: ' 34px',
-  fontFamily: 'Inter, sans serif',
-  fontStyle: 'normal',
-  fontWeight: 500,
-  fontSize: '14px',
-  lineHeight: '18px',
-  width: '100%',
-  marginRight: '12px',
-  flexGrow: 1,
-  padding: '8px 25.75px',
-
-  [lightTheme.breakpoints.up('table_834')]: {
-    padding: '8px 43.25px',
+const InternalLinkButtonStyled = styled(InternalLinkButton)(() => ({
+  borderRadius: 6,
+  height: 32,
+  padding: '4px 13.5px 4px 13.5px',
+  ':hover': {
+    padding: '4px 13.5px 4px 13.5px',
   },
-});
+  '& div': {
+    letterSpacing: '-2%',
```

Comment:
letter spacing can't be in `%`

Reply:
Are you sure, Can you check in here and let me know
https://www.scaler.com/topics/letter-spacing-css/